nlp/front/shared/src/main/kotlin/config/ApplicationDefinition.kt

```diff
@@ -70,6 +70,10 @@ data class ApplicationDefinition(
      * Unknown intent threshold level.
      */
     val unknownIntentThreshold: Double = 0.0,
+    /**
+     * Known intent threshold level.
+     */
+    val knownIntentThreshold: Double = 0.0,
```

Comment:
Default value should be the original value 0.1 to prevent any breaking changes, we need to make sure that this will not impact existing applications.